### PR TITLE
Add the support of mfcc feature for DS2

### PR DESCRIPTION
**Pull request by kuke**
Add mfcc feature for audio data, and the training of model is in progress:

### CODE REVIEW

File: deep_speech_2/README.md [hunk 1]
```diff
@@ -38,7 +38,13 @@ python datasets/librispeech/librispeech.py --help
 python compute_mean_std.py
 ```
 
-`python compute_mean_std.py` computes mean and stdandard deviation for audio features, and save them to a file with a default name `./mean_std.npz`. This file will be used in both training and inferencing.
+`python compute_mean_std.py` computes mean and stdandard deviation for audio features, and save them to a file with a default name `./mean_std.npz`. This file will be used in both training and inferencing. The default feature of audio data is power spectrum, currently the mfcc feature is also supported. To train and infer based on mfcc feature, you can regenerate this file by
```
Comment (pkuyym) [inline deep_speech_2/README.md]: ```currently the mfcc feature is also supported```，changing ```currently``` to ```and```  should be better? There's no need to tell the user that mfcc is added just now.

Comment (pkuyym) [reply deep_speech_2/README.md]: ```you can regenerate this file by```，why ```regenerate``` when first running ds2?

File: deep_speech_2/requirements.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ wget==3.2
 scipy==0.13.1
 resampy==0.1.5
 https://github.com/kpu/kenlm/archive/master.zip
+python_speech_features
```
Comment (pkuyym) [inline deep_speech_2/requirements.txt:5]: Add a version number.

**Review by pkuyym — COMMENTED** [review]
Almost LGTM

File: deep_speech_2/README.md [hunk 3]
```diff
@@ -38,7 +38,13 @@ python datasets/librispeech/librispeech.py --help
 python compute_mean_std.py
 ```
 
-`python compute_mean_std.py` computes mean and stdandard deviation for audio features, and save them to a file with a default name `./mean_std.npz`. This file will be used in both training and inferencing.
```
Comment (xinghai-sun) [inline deep_speech_2/README.md:41]: `python compute_mean_std.py` computes  --> "It will compute"

File: deep_speech_2/README.md (hunk 1 above)
Comment (xinghai-sun) [inline deep_speech_2/README.md]: 1. you can regenerate --> please regenerate
2.spectrum or spectrogram ?

File: deep_speech_2/README.md [hunk 4]
```diff
@@ -38,7 +38,13 @@ python datasets/librispeech/librispeech.py --help
...
+`python compute_mean_std.py` computes mean and stdandard deviation for audio features, and save them to a file with a default name `./mean_std.npz`. This file will be used in both training and inferencing. The default feature of audio data is power spectrum, currently the mfcc feature is also supported. To train and infer based on mfcc feature, you can regenerate this file by
+
+```
+python compute_mean_std.py --specgram_type mfcc
+```
+
+and specify the ```specgram_type``` to ```mfcc``` in each step, including training, inference etc.
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: 1. “in each step, including training, inference etc.” --》 “ when running train.py, infer.py, evaluator.py or tune.py"
2. ```specgram_type``` to ```mfcc``` --> ```--specgram_type mfcc```

**Review by xinghai-sun — APPROVED** [review]
Great. Looking forward to better experimental results with MFCC.

File: deep_speech_2/requirements.txt (hunk 2 above)
Comment (kuke) [reply deep_speech_2/requirements.txt:5]: No version number

File: deep_speech_2/README.md (hunk 3 above)
Comment (kuke) [reply deep_speech_2/README.md:41]: Done

File: deep_speech_2/README.md (hunk 1 above)
Comment (kuke) [reply deep_speech_2/README.md]: Done

File: deep_speech_2/README.md (hunk 4 above)
Comment (kuke) [reply deep_speech_2/README.md]: Done

File: deep_speech_2/README.md (hunk 1 above)
Comment (kuke) [reply deep_speech_2/README.md]: Done

**Pull request merged**